src/cart/crt_corpc.c

```diff
@@ -899,20 +904,28 @@ crt_corpc_req_hdlr(struct crt_rpc_priv *rpc_priv)
 	}
 
 forward_done:
-	if (rc != 0 && rpc_priv->crp_flags & CRT_RPC_FLAG_CO_FAILOUT)
-		co_failout = true;
+	if (rc != 0) {
```

Comment:
seems we should just move crt_corpc_fail_parent_rpc() here, instead of doing it in each error case above